examples/wakaama/README.md

```diff
@@ -0,0 +1,174 @@
+Eclipse Wakaama Example Client
+==============================
+
+This application is a port of the Eclipse Wakaama [example client
+code](https://github.com/eclipse/wakaama/tree/master/examples/client)
+to RIOT/GNRC.
+
+Usage
+=====
+
+Edit the defines in `lwm2mconfig.h` according to your setup (see the
+comments for what they do).
+
+Build, flash and start the application:
+```
+export BOARD=your_board
+make
+make flash
+make term
+```
+
+(Or Skip `make flash` if you're building for `BOARD=naive`.)
+
+The `term` make target starts a terminal emulator for your board. It
+connects to a default port so you can interact with the shell, usually
+that is `/dev/ttyUSB0`. If your port is named differently, the
+`PORT=/dev/yourport` (not to be confused with the UDP port) variable can
+be used to override this.
+
+
+Available LwM2M Objects
+=======================
+
+At this time, only the mandatory objects for a client are present. The
+*server*, *security* and *access control* objects are taken as is from
+the example client in the Wakaama source tree. The *device* object was
+re-implemented for RIOT (see `object_device.c`).
+
+Adding more objects should be as easy as implmenting a "contructor"
+and then incrementing the `OBJ_COUNT` in `lwm2mclient.c` and extending
+`run_server()` to call the constructor and adding the instance(s) to
+`objArray`.
+
+
+LwM2M Test Server
+=================
+
+[Eclipse Leshan](https://github.com/eclipse/leshan#test-leshan-demos-locally)
+is good for local testing. Just run it on the link local address of
+the bridge interface created by `dist/tools/tapsetup/tapsetup` and add
+the credentials set in `lwm2mconfig.h` via the web interface. Since the
+example bootstrap server doesn't seem to support setting DTLS credentials
+for the server itself via the web interface, you'll have to create a
+new client config, edit `data/bootstrap.json` and set `securityMode`,
+`publicKeyOrId`, `secretKey` and maybe `serverPublicKey` and restart
+the server.
+
+The LwM2M demo server supports setting DTLS credentials on the *Security*
+tab.
+
+Example (Linux):
+```
+wget https://hudson.eclipse.org/leshan/job/leshan/lastSuccessfulBuild/artifact/leshan-server-demo.jar
+
+ADDR=$(ip address show dev tapbr0 | grep inet6 |\
+   head -1 | awk '{print $2}' | cut -d'/' -f1)
+
+java -jar ./leshan-server-demo.jar \
+   --coapshost ${ADDR} \
+   --coaphost ${ADDR}
+```
+
+
+Example output
+==============
+
+When registering directly to an LwM2M server, with DTLS, without boostrapping:
```

Comment:
You're right. I think I had that set by default, so that the code compiled as is would match the example output, but I might have commented it out while testing things and forgotten to put it back in before committing a change.

I'll specify which debug settings to use to get a matching debug output in README.md.